[Docs] Fix cookbook pattern for AI SDK (#2099)
vercel/workflow: docs/content/docs/v4/cookbook/integrations/ai-sdk.mdx

title: AI SDK
description: Use AI SDK's streamText directly inside durable workflows for lower-level control over model calls and tool execution.
type: guide

> summary: Use streamText() inside a workflow for full control over model options, stop conditions, and output schemas. The per-turn step is durable; individual tool calls inside it are not.

Reviewer: ```suggestion
summary: Use streamText() inside a workflow for full control over model options, stop conditions, and output schemas. Each turn is durable; individual tool calls and LLM calls inside it are not.
```

  schema: z.object({ message: z.string() }),
});

// Tools are plain async functions in this pattern. `streamText` calls them

> // from inside `runTurn` (a step), and the `"use step"` directive is a no-op

Reviewer: I feel like this comment is more of a changelog than describing the current state? Since we removed the "use step"

Author: Good call — reworded to describe how tools run inside `runTurn` without referencing removed `"use step"` directives.
